MapboxCoreNavigation/CLLocation.swift

```diff
@@ -0,0 +1,104 @@
+import CoreLocation
+
+extension Array {
+    
+    /**
+     Initializes a [CLLocation] from a JSON string at a given filePath.
+     
+     The JSON string must conform to the following structure:
+     [{
+         "latitude": 37.8,          // latitude or lat
+         "longitude": -122.4        // longitude, lng, or lon
+         "verticalAccuracy": 4,
+         "speed": 21.0
+         "horizontalAccuracy": 5,
+         "course": 0.48
+         "timestamp": 1497475447,   // timestamp as unix timestamp or ISO8601Date
+         "altitude": 57.26
+     }]
+     
+     - parameter filePath: The file’s path.
+     - returns: A [CLLocation].
+     */
+    public static func locations(from filePath: String) -> [CLLocation]! {
```

Comment:
I think if this is going to be public and used outside of this file, it should be moved to it's own `Array.swift` just so we can stay organized/predictable.